docs/IE_PLUGIN_DG/layout.xml

```diff
@@ -14,7 +14,6 @@
             <tab type="user" url="@ref plugin_build" visibile="yes" title="Build Your Plugin with CMake*"/>
             <tab type="user" url="@ref plugin_testing" visibile="yes" title="Test Your Plugin"/>
             <tab type="user" url="@ref quantized_networks" visibile="yes" title="Quantized networks guide"/>
-            <tab type="user" url="@ref new_ngraph_transformation" visibile="yes" title="Writing ngraph transformations"/>
```

Comment:
why is it removed?

Reply:
I moved transformation document to nGraph section. But I also updated all links.